src/interfaces/src/plugin/data/interfaces/interface/ipv4/address.c

```diff
@@ -79,6 +81,56 @@ int interfaces_interface_ipv4_address_element_set_ip(interfaces_interface_ipv4_a
     return 0;
 }
 
+int interfaces_interface_netmask_to_prefix_length(const char *netmask)
```

Comment:
Such a function already exists: https://github.com/telekom/sysrepo-plugin-interfaces/blob/refactor/src/interfaces/src/plugin/data/interfaces/interface/ipv4/address.h#L31

Reply:
Also a note, its best to not use netmask when using load API because of concistency - libnl uses only prefix-length. Netmask is only used on changes when a specific change is iterated which uses YANG netmask leaf instead of the prefix-length and when using store API and a netmask is found inside the datastore instead of the prefix-length.